apps/financial-aid/backend/src/app/modules/municipality/municipality.controller.ts

```diff
@@ -26,13 +26,16 @@ import {
   CreateMunicipalityDto,
 } from './dto'
 import { CreateStaffDto } from '../staff/dto'
+import { MunicipalitiesFinancialAidScope } from '@island.is/auth/scopes'
 
 @UseGuards(IdsUserGuard)
 @Controller(`${apiBasePath}/municipality`)
 @ApiTags('municipality')
 export class MunicipalityController {
   constructor(private readonly municipalityService: MunicipalityService) {}
 
+  @UseGuards(ScopesGuard)
+  @Scopes(MunicipalitiesFinancialAidScope.read)
```

Comment:
ditto on last comment.

Reply:
The municipality data is not sensitive. At least for the time being. It has the following columns: 

`id`, `municipality_id`, `name`, `homepage`, `active`, `individual_aid_id`, `cohabitation_aid_id`, `email` and `rules_homepage`

This data is then used to see if user can apply for financial aid in the applicants municipality and to show data relevant to that municipality. 

This endpoint is for example called after the applicant approves data gathering to see if their municipality supports financial aid.

With that in mind, do you feel it should be better guarded?

Reply:
I did my initial report based on who was supposed to call the endpoints. If I remember correctly, only web-veita was supposed to call this. That's why I flagged it in the report.

So it's up to you :).

Reply:
This is called from `useMunicipality` hook: https://github.com/island-is/island.is/blob/3666ae66b8c6e0b02d3f91f1da39f151dc552c74/libs/financial-aid/shared/src/components/hooks/useMunicipality.ts 

Which is used in both clients, will resolve conversation but feel to unresolve if you feel the need :)